### PR TITLE
Iosxe/show class map

### PR DESCRIPTION
**Pull request by reustaquio22**
Parser for IOSXE Show Class Map

### CODE REVIEW

**Comment by karmoham** [comment]
Hi Roy,

Thanks for your contribution!

Regards,
Karim

File: src/genie/libs/parser/iosxe/show_class_map.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+
+# ===================
+# Parser for:
+#  * 'show class-map'
+# ===================
+class ShowClassMap(ShowClassMapSchema):
+    """Parser for show class-map"""
+
+    cli_command = ['show class-map']
```
[the remark this answers is not in the document]
Comment (reustaquio22) [reply src/genie/libs/parser/iosxe/show_class_map.py]: This has been updated.

File: src/genie/libs/parser/iosxe/show_class_map.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        cm_header_capture = re.compile(
+            r"^(\s+|)Class\sMap\s(?P<match_criteria>\S+)\s+(?P<cm_name>\S+)\s+\(id\s+(?P<cm_id>\d+)\)", re.MULTILINE)
+        description_capture = re.compile(r"^Description:\s+(?P<description>.*$)", re.MULTILINE)
+        match_policy_capture = re.compile(r"^Match\s+(?P<match_policy>\S+)", re.MULTILINE)
+
+
+        # Remove unwanted lines from raw text
+        for line in out.splitlines():
+            line = line.strip()
+            cm_match = cm_header_capture.match(line)
```
[the remark this answers is not in the document]
Comment (reustaquio22) [reply src/genie/libs/parser/iosxe/show_class_map.py]: I added empty lines between sections.

File: changelog/undistributed.rst [hunk 3]
```diff
@@ -10,8 +10,8 @@
 --------------------------------------------------------------------------------
 
 * IOSXE
-    * Added ShowStackwiseVirtual
```
[the remark this answers is not in the document]
Comment (reustaquio22) [reply changelog/undistributed.rst:13]: This has been normalized.

**Pull request merged**